Brewfile

```diff
@@ -1,5 +1,6 @@
-# 2021-02-08 - The CI checks the hash of this file to determine if to create a new
-# cache or not. If you want to force a new cache simply change the date on the line above
+# required to run devservices
+cask 'docker'
```

Comment:
I'm moving it to the top since there's a chance that there is a race condition writing files in `/Applications/Docker.app` when brew installs it and before we call `open`. Call me paranoid. I want to reduce the chances that the intermittent issue is actually that.

Reply:
lol

Reply:
It makes me want to curl in a corner 😄